src/components/_common/overview-filters/text-filter/OverviewTextFilter.tsx

```diff
@@ -36,25 +35,9 @@ export const OverviewTextFilter = ({ hideLabel }: Props) => {
         [dispatch]
     );
 
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    const logInputToSentry = useCallback(
-        debounce((value: string) => {
-            if (value.length < 3) {
-                return;
-            }
-
-            Sentry.captureMessage(
-                `Oversiktsside fritekst input: "${value}"`,
-                'info'
-            );
-        }, 3000),
-        []
-    );
-
```

Comment:
Kan vi erstatte denne loggingen med tilsvarende fra Faro? Vi får så vidt jeg vet fortsatt ikke lov til å logge fritekst input til Amplitude, som var grunnen til at Sentry ble brukt for akkurat denne 😅

Reply:
Jepp! Sender inn som en custom event. Har sjekket og bekreftet at den kommer inn som en event via Faro til Loki nå. Vi må diskutere litt hvordan vi ønsker å sette opp disse panelene, men dataene kommer inn ihvertfall.